Cirrious.FluentLayout/FluentLayout.cs

```diff
@@ -153,7 +154,7 @@ private void ThrowIfSecondItemAlreadySet()
 
         public IEnumerable<NSLayoutConstraint> ToLayoutConstraints()
         {
-            var constraint = NSLayoutConstraint.Create(
+            Constraint = NSLayoutConstraint.Create(
```

Comment:
Do you mean `ToLayoutConstraints()[0]`?
We could try removing this method and add `ToLayoutConstraint()`but then other parts of the code needs some changes. I¨m not sure about the reason behind returning an array when there is a single constraint